go.mod

```diff
@@ -1,82 +1,14 @@
+// SPDX-FileCopyrightText: 2024 Serpent OS Developers
+// SPDX-License-Identifier: MPL-2.0
+
 module github.com/der-eismann/libstone
```

Comment:
Could you replace this with `github.com/serpent-os/libstone-go` please?

Reply:
Fine for me! I always feel a bit conflicted because IMHO repo name should match package name, but at the same time it should be clear that this is go, not rust.